src/public/js/likeToggle.js

```diff
@@ -0,0 +1,34 @@
+    document.addEventListener('DOMContentLoaded', function () {
+    const likeForms = document.querySelectorAll('.like-form');
+    const unlikeForms = document.querySelectorAll('.unlike-form');
+
+    likeForms.forEach(function (likeForm) {
+        likeForm.addEventListener('submit', function (e) {
+            e.preventDefault(); // Prevent the default form submission
```

Comment:
remove comments

Reply:
Comment is resolved, restructured the js with AJAX. Thank you 🙇🙇